### PR TITLE
tests: Use Runner.get_library consistently. NFC.

### PR DESCRIPTION
**Pull request by sbc100**
The `configure` argument is supposed to be the thing we
pass to `emconfigure` not `emconfigure` itself.

Also, remove the extra argument to build_library routine.

I'm hoping to follow this up a refactor that has us
call `emconfigure` from the outside rather than using
`building.configure`.

### CODE REVIEW

File: tests/runner.py [hunk 1]
```diff
@@ -1622,11 +1622,11 @@ def build_library(name,
         with open(os.path.join(project_dir, 'configure_err'), 'w') as err:
           stdout = out if EM_BUILD_VERBOSE < 2 else None
           stderr = err if EM_BUILD_VERBOSE < 1 else None
-          building.configure(configure + configure_args, env=env,
+          building.configure(configure, env=env,
                              stdout=stdout,
                              stderr=stderr,
                              cwd=project_dir)
-    except subprocess.CalledProcessError:
+    except Exception:
```
Comment (aheejin) [inline tests/runner.py:1629]: Why the change?

File: tests/runner.py [hunk 2]
```diff
@@ -1121,9 +1123,8 @@ def get_poppler_library(self, env_init=None):
   def get_zlib_library(self):
     if WINDOWS:
       return self.get_library(os.path.join('third_party', 'zlib'), os.path.join('libz.a'),
-                              configure=[path_from_root('emconfigure.bat')],
-                              configure_args=['cmake', '.'],
-                              make=['mingw32-make'],
+                              configure=['cmake', '.'],
+                              make=['cmake', '--build', '.'],
```
Comment (aheejin) [inline tests/runner.py:1127]: Is this change intended? (Before it was mingw..)

**Review by aheejin — APPROVED** [review]

File: tests/runner.py (hunk 2 above)
Comment (sbc100) [reply tests/runner.py:1127]: Yes, its actually better to call `cmake --build` because then it will invoke what ever the chosen build system was.   Calling `mingw32-make` is more fragile, and in the case the `mingw32-make` was actually used by cmake it is equivalent.

File: tests/runner.py (hunk 1 above)
Comment (sbc100) [reply tests/runner.py:1629]: I ill revert that this part since I'm planning a followup that will change this anyway.
